e2e/fixtures/authenticatedUserPages.ts

```diff
@@ -0,0 +1,141 @@
+import { test as base, Browser, expect, type Page } from '@playwright/test'
+import { unlinkSync } from 'fs'
+import { baseURL } from '../../playwright.config'
+import {
+  cleanupTestUser,
+  createTestUser,
+  getAdminPAT,
+  getUserIdFromLocalStorage,
+  loginTestUser,
+} from '../helpers/testUser'
+import {
+  storageStateDir,
+  userConfigs,
+  userPrefix,
+  UserPrefixes,
+  userValidatedPrefix,
+} from '../helpers/userConfig'
+import { waitForInitialPageLoad } from '../helpers/utils'
+
+type AuthenticatedUserPageTestFixtures = {
+  userPage: Page
+  validatedUserPage: Page
+}
+
+type StorageStatePaths = { [key in UserPrefixes]?: string }
+type AuthenticatedUserPageWorkerFixtures = {
+  createUsers: void
+  storageStatePaths: StorageStatePaths
+}
+
+const createPage = async (browser: Browser) => {
+  const page = await browser.newPage()
+  // specify baseURL so we call this method outside of a test
+  // ...i.e. where the baseURL test fixture is not available
+  await page.goto(baseURL)
+  await waitForInitialPageLoad(page)
+  return page
+}
+
+// Generate test fixture for a specified user
+// ...based on: https://github.com/microsoft/playwright/issues/14570
+function createUserPageFixture<Page>(userPrefix: UserPrefixes) {
+  return async (
+    { browser, storageStatePaths },
+    use: (r: Page) => Promise<void>,
+  ) => {
+    const context = await browser.newContext({
+      storageState: storageStatePaths[userPrefix],
+    })
+    const validatedUserPage = await context.newPage()
+    await use(validatedUserPage)
+    await context.close()
+  }
+}
+
+export const testAuth = base.extend<
```

Comment:
Use `testAuth` for these fixtures to distinguish between unauthenticated and authenticated tests in the same test file.